site_scons/buildutils.py

```diff
@@ -472,137 +591,105 @@ def add_RegressionTest(env):
         action=env.Action(regression_test, regression_test_message))
 
 
-def quoted(s):
-    """ Returns the given string wrapped in double quotes."""
-    return '"%s"' % s
-
-
 def mglob(env, subdir, *args):
```

Comment:
This could use a better name, I recall that at some point it wasn't obvious that this was a cantera-specific command. And PEP 484, of course ...